src/commands/branch.rs

```diff
@@ -65,7 +65,14 @@ pub struct BranchDeleteArgs {
 /// preceded by a "+". For information about branches, see
 /// https://github.com/martinvonz/jj/blob/main/docs/branches.md.
 #[derive(clap::Args, Clone, Debug)]
-pub struct BranchListArgs;
+pub struct BranchListArgs {
+    /// Show branches whose local targets are in the given revisions.
+    ///
+    /// If specified, deleted/forgotten branches aren't listed since they have
```

Comment:
I would say:

> Deleted/forgotten branches cannot be shown with this option since they have no local targets.

"If specified" seems a little misleading, since AFAICT there's no longer a way to specify them.

Reply:
Ah, I realized what I was missing as I read the test. Perhaps a better way to say this is:

> Note that `-r deleted_branch` will not work since `deleted_branch` wouldn't have a local target.

Reply:
Much better. Thanks.